system/zutty/zutty.info

```diff
@@ -1,8 +1,8 @@
 PRGNAM="zutty"
-VERSION="0.14"
+VERSION="0.15"
 HOMEPAGE="https://tomscii.sig7.se/zutty/"
-DOWNLOAD="https://github.com/tomscii/zutty/archive/0.14/zutty-0.14.tar.gz"
-MD5SUM="45c3a53e371ffbb87027d41d99799989"
+DOWNLOAD="https://git.hq.sig7.se/zutty.git/snapshot/0.15.tar.gz"
```

Comment:
Just leaving a note related to Willy email. The slackbuild needs to also be able to handle the source if people download it from the links on the website with their browser. (I don't know if this git server can also return a dedicated filename like github can).

Currently this link gives the filename zutty-0.15-3fb1516.tar.gz if you use the link. Can you make the slackbuild handle both please?